NAMESPACE

```diff
@@ -1,6 +1,9 @@
 # Generated by roxygen2: do not edit by hand
 
+export("%>%")
```

Comment:
Just out of curiosity, how did you generate these new lines, e.g. export("%>%") and importFrom(xxx) ?

Reply:
> Just out of curiosity, how did you generate these new lines, e.g. export("%>%") and importFrom(xxx) ?

I think there is a video (524-lecture-4-addendum) on canvas tells us to use use_package() and use_pipe()

Reply:
Yep, like what @lirnish said 👆